src/db/collections/analytics/influx.rs

```diff
@@ -61,6 +61,9 @@ impl From<Measurement> for influxdb::WriteQuery {
                 .add_field("total_key_bytes", m.inner.total_key_bytes)
                 .add_field("total_data_bytes", m.inner.total_data_bytes)
                 .add_field("total_storage_deposit_value", m.inner.total_storage_deposit_value),
+            Measurement::MilestoneSizeAnalytics(m) => m
+                .prepare_query("stardust_milestone_size")
+                .add_field("total_milestone_bytes", m.inner.total_milestone_bytes),
```

Comment:
Would it make sense to compute this per payload type? Both tagged data payloads and outputs allow adding data to the tangle.

Reply:
We could certainly do that, but the question is whether there's something interesting to draw conclusions from that analytic. Can you come up with something? I think I'ld not add this preemptively but rather change it when there's an actual request/demand for it.

Reply:
I think it's interesting to see. This way we could for example try to gauge if people start using the ledger to store more and more information (remember we also push number of payloads, so we can compute averages in InfluxDb).

Reply:
Also, if we change it later we have to resync.

Reply:
right, okay I'll add it.